components/server/src/installation-admin/installation-admin-controller.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { injectable, inject } from 'inversify';
+import * as express from 'express';
+import { InstallationAdminDB } from '@gitpod/gitpod-db/lib';
+
+@injectable()
+export class InstallationAdminController {
+    @inject(InstallationAdminDB) protected readonly installationAdminDb: InstallationAdminDB;
+
+    get apiRouter(): express.Router {
+        const router = express.Router();
+
+        router.get('/data', async (req: express.Request, res: express.Response) => {
+            const data = await this.installationAdminDb.getData();
+
+            res.send(data);
```

Comment:
I feel a bit uncomfortable with simply sending the whole record over this unprotected endpoint. If someone adds additional properties to the database record, the person needs to have in mind that it will be exposed here. Maybe we filter the data at this point and expose only the data that is needed by the telemetry component? What do you think? Are my concerns exaggerated?

We could also create a `getTelemetryData(...)` function instead and let `InstallationAdminDB` filter the stuff for us.

Reply:
Yeah, I'm fine with that.

Concerns are probably a little exaggerated, but I'd rather be safe than sorry so I'll change to `getTelemetryData`

Reply:
I don't think we should start filtering here. Instead, I propose we either:
 - make the entire endpoint non-public (see comment in server.ts)
 - use the DB as interface in the first place (not sure whether this is actually a good idea)

If we keep the endpoint, using `res.status(200).json(data);` will set correct content type header as well. :+1: